policy/modules/services/console_setup.fc

```diff
@@ -0,0 +1,11 @@
+/etc/console-setup	-d	gen_context(system_u:object_r:console_setup_conf_t,s0)
+/etc/console-setup/.*	gen_context(system_u:object_r:console_setup_conf_t,s0)
```

Comment:
These 2 lines seem to be merged together:
```
/etc/console-setup(/.*)?	gen_context(system_u:object_r:console_setup_conf_t,s0)
```

This also applies to `/run/console-setup` below.